client/src/pages/resources/styles.scss

```diff
@@ -1,6 +1,18 @@
 @import '../../styles/colors.scss';
 
 .resources-page {
+  .calendly-button {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    border: 1px solid #5a72e1;
+    box-sizing: border-box;
+    border-radius: 5px;
+    background-color: transparent;
+    color: #5a72e1;
+    position: absolute;
+    right: 3.78%;
```

Comment:
This is not responsive. Look into how to flex-grow to get this into the position you would like.